staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -135,7 +135,7 @@ func (f *SimpleRestOptionsFactory) GetRESTOptions(resource schema.GroupResource)
 		Decorator:               generic.UndecoratedStorage,
 		EnableGarbageCollection: f.Options.EnableGarbageCollection,
 		DeleteCollectionWorkers: f.Options.DeleteCollectionWorkers,
-		ResourcePrefix:          f.Options.StorageConfig.Prefix + "/" + resource.Group + "/" + resource.Resource,
```

Comment:
ignoring the prefix in general feels strange. If it is empty, it is what I would expect. But if I set `StorageConfig.Prefix` to non-empty?